src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -58,10 +58,10 @@ public object[] NestedModules
         private object[] _nestedModules;
 
         /// <summary>
-        /// Set the GUID in the manifest file.
+        /// Gets or sets the GUID in the manifest file.
         /// </summary>
         [Parameter]
-        public Guid Guid
+        public Guid GUID
         {
             get { return _guid; }
```

Comment:
@pougetat You could review all such unneeded private fields and remove them. Better in follow PR.

Reply:
sounds good